.nycrc

```diff
@@ -0,0 +1,9 @@
+{
+  "reporter": [
+    "lcov"
+  ],
+  "extension": [
+    ".jsx"
```

Comment:
Yeah, it always covers `.js` files. `"extension"` just provides _additional_ extensions to instrument. I agree, it's weird.

I just confirmed locally also by using the `"text"` reporter for easily visualizing the files covered. Here's a snippet that shows both `.js` and `.jsx` are included:

<img width="597" alt="screen shot 2017-08-11 at 9 37 14 am" src="https://user-images.githubusercontent.com/697848/29217917-c22f35ee-7e78-11e7-8bdd-08a6c2317a2b.png">